test/requests/rename_test.rb

```diff
@@ -0,0 +1,47 @@
+# typed: true
+# frozen_string_literal: true
+
+require "test_helper"
+
+class RenameTest < Minitest::Test
```

Comment:
Looks like we're not testing the file renaming here?

Reply:
Added behaviour to test the renaming.